src/librustdoc/html/static/main.js

```diff
@@ -527,10 +527,10 @@ function hideThemeButtonState() {
         hideThemeButtonState();
     }
 
-    var disableShortcuts = getSettingValue("disable-shortcuts") === "true";
+    var disableShortcuts = !!getSettingValue("disable-shortcuts");
```

Comment:
In this case however, please keep the `=== true`, I find the new code quite hard to read. Also, for strings comparison (even though in here it doesn't really matter), `===` is much faster.

Reply:
👍🏻 

I agree it's hard to read initially, though FWIW in many JS codebases, `!!` is idiomatic to coerce a potentially truthy or falsy value to a definite `true` or `false` boolean. For instance that Airbnb guide recommends it: https://airbnb.io/javascript/#coercion--booleans

That said, in this case I used it quite wrongly, since "false" (or any nonempty string) is a truthy value, so this would coerce `"false"` to `true`. Oops. 😳 Reverted.